packages/javascript/.changesets/update-types-dependency-and-remove-params-restrictions.md

```diff
@@ -0,0 +1,6 @@
+---
+bump: "patch"
+type: "change"
+---
+
+Update types dependency and remove params restrictions
```

Comment:
I think a changelog entry is automatically added by Mono for the dependency update.

```suggestion
Remove a restriction in our `setParams` TypeScript types that did not allow nested objects to be sent as params. It is now possible to send nested objects and arrays as values inside the params object.
```